.buildkite/pipeline.yml

```diff
@@ -12,27 +12,48 @@
 # permissions and limitations under the License.
 
 steps:
-  - label: ':go: go mod download'
-    command: 'go mod download'
 
-  # We use a "wait" step here, because Go's module logic freaks out when
-  # multiple go builds are downloading to the same cache.
-  - wait
+  - label: ":docker: Build"
+    env:
+      DOCKER_BUILDKIT: "1"
+    command:
+      >
+        docker build \
+          --progress=plain \
+          -f tools/docker/Dockerfile \
```

Comment:
Can we use long flags consistently?  `--file` and `--tag` instead of `-f` and `-t` respectively.